component/main.js

```diff
@@ -1,5 +1,17 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import ClientCore from "./client/ClientCore.jsx";
+import * as firebase from "firebase";
+
+// Initialize Firebase
+var config = {
+  apiKey: "AIzaSyBakHeV8lMlysuBRtIWU9vz_hv6dF_zHxM",
+  authDomain: "url-connet.firebaseapp.com",
+  databaseURL: "https://url-connet.firebaseio.com",
+  projectId: "url-connet",
+  storageBucket: "url-connet.appspot.com",
+  messagingSenderId: "1089725560944"
+};
```

Comment:
No, those are specific to this app. There is no way to look up any of those. This is how firebase is designed to work.